<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalculator</class>
 <widget class="QMainWindow" name="CreditCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>514</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Credit calculator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/icon.png</normaloff>../assets/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,1" columnstretch="0,1,0">
    <item row="1" column="0">
     <widget class="QLabel" name="term_label">
      <property name="text">
       <string>Term</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QDoubleSpinBox" name="sum_input">
      <property name="showGroupSeparator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="suffix">
       <string> RUB</string>
      </property>
      <property name="maximum">
       <double>1000000000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>500.000000000000000</double>
      </property>
      <property name="value">
       <double>1000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="term_input">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QRadioButton" name="credit_type_differentiated_radio">
      <property name="text">
       <string>Differentiated</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QTableWidget" name="result_table">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Monthly payment</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>In debt</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>In interest</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Debt rest</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="sum_label">
      <property name="text">
       <string>Sum</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="term_type_input">
      <item>
       <property name="text">
        <string>Months</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Years</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="rate_label">
      <property name="text">
       <string>Rate</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="rate_input">
      <property name="suffix">
       <string> %</string>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.010000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QRadioButton" name="credit_type_annuity_radio">
      <property name="text">
       <string>Annuity</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="credit_type_label">
      <property name="text">
       <string>Credit type</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="rate_input_type">
      <item>
       <property name="text">
        <string>Per annum</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Per month</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QPushButton" name="calculate_button">
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>514</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionMath_and_plots_calculator"/>
    <addaction name="actionDeposit_calculator"/>
   </widget>
   <addaction name="menuMode"/>
  </widget>
  <action name="actionMath_and_plots_calculator">
   <property name="text">
    <string>Math and plots calculator</string>
   </property>
  </action>
  <action name="actionDeposit_calculator">
   <property name="text">
    <string>Deposit calculator</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>sum_input</tabstop>
  <tabstop>term_input</tabstop>
  <tabstop>term_type_input</tabstop>
  <tabstop>rate_input</tabstop>
  <tabstop>rate_input_type</tabstop>
  <tabstop>credit_type_annuity_radio</tabstop>
  <tabstop>credit_type_differentiated_radio</tabstop>
  <tabstop>calculate_button</tabstop>
  <tabstop>result_table</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
